scene/gui/spin_box.cpp

```diff
@@ -392,4 +393,4 @@ SpinBox::SpinBox() {
 	range_click_timer = memnew(Timer);
 	range_click_timer->connect("timeout", callable_mp(this, &SpinBox::_range_click_timeout));
 	add_child(range_click_timer, false, INTERNAL_MODE_FRONT);
-}
+}
```

Comment:
```suggestion
}

```
Missing line termination, though I don't know why you edited this line like this manually

Reply:
I accidentally deleted the file contents because I had the wrong window focused and had to copy paste it back, my bad